### ISSUE TITLE
Ampersand being changed to 'x' from a template

### ISSUE THREAD

**Issue by bogeorge** [dtjohnson/xlsx-populate#125]
Has anyone else encountered an issue where a template that contains cells with Xml based characters getting turned into 'x'?  I did some tests and it only seems like &, <, > have this happen but not ' or ".  Doing some digging into the code (without debugging so far) and I'm still not sure if it's JSZip.loadAsync doing it or if it's after inside of SharedStrings.

**Comment by dtjohnson** [comment]
Is this in the browser version? I'm not seeing this issue in Node...

**Comment by bogeorge** [comment]
Yes this is the browser version.  I'm using XlsxPopulate.fromDataAsync(fileData) where my file data is an Array Buffer returned from a Excel Template file in my application.

**Comment by dtjohnson** [comment]
Please try with v1.17.0. There is also a non-minified version available now.
